backend/routes/questionRoutes.js

```diff
@@ -3,12 +3,12 @@ const { getAllQuestions, createQuestion, deleteQuestion } = require('../controll
 const router = express.Router();
 
 // Route to get all questions (homepage)
-router.get('/', getAllQuestions);
+router.get('/get-all-questions', getAllQuestions);
 
 // Route to create a new question
 router.post('/create-question', createQuestion);
 
 // Route to delete a question
-router.delete('/', deleteQuestion);
+router.delete('/delete-question', deleteQuestion);
```

Comment:
Same as above, changing the path would likely mean that our frontend paths and backend paths won't be a 1 to 1 matching (because it is unlikely the frontend path will be "/delete-questions" but rather a delete request sent at the same path the user was in, like "/", for example)

This is owing to our discussion that our deletion of questions will be done via admin account being able to click a delete button to the right of each question (so on the frontend side, the path did not change).

Reply:
Personally I feel that the frontend route is designed for navigation and user experience while the backend route is descriptive to allow greater clarity on what function is being called.

Having both get/post/delete mapped to the same route is confusing considering that the function may be used at other paths not just limited to `/` on the frontend path.

i.e. I could call delete on '/questions' or perhaps '/dashboard' for admin etc etc

Reply:
Having the backend route to be descriptive seems like a good approach to take. Good suggestion!